ethereum/listener.go

```diff
@@ -7,21 +7,26 @@ package ethereum
 
 import (
 	"context"
-	"github.com/Shopify/sarama"
 	"math/big"
 	"time"
 
+	"github.com/Shopify/sarama"
+
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/ethereum/go-ethereum/ethclient"
+
 	"github.com/persistenceOne/persistenceBridge/application/db"
 	"github.com/persistenceOne/persistenceBridge/application/shutdown"
 	"github.com/persistenceOne/persistenceBridge/kafka/utils"
 	"github.com/persistenceOne/persistenceBridge/utilities/logging"
 )
 
+const finality = 12
```

Comment:
should be in constants, it's already v0.6.0

Reply:
it is a local private constant. Could you tell me more why it worth it to keep it in a separate package and increase visibility to  public?